src/client/src/error.rs

```diff
@@ -26,12 +26,7 @@ pub enum Error {
         backtrace: Backtrace,
     },
 
-    #[snafu(display(
-        "Failed to do Flight get, addr: {}, code: {}, source: {}",
-        addr,
-        tonic_code,
-        source
-    ))]
+    #[snafu(display("Failed to do Flight get, code: {}, source: {}", tonic_code, source))]
     FlightGet {
         addr: String,
```

Comment:
this field can be removed as well

Reply:
We can keep this field for internal logging

Reply:
I wanted to delete this field before. Then I thought I might need to use it in the error field as well.